### PR TITLE
Fix/youtube cards mobile

### PR DESCRIPTION
**Pull request by RajVerma97**
Polish the mobile view for Youtube Cards #587

### MOBILE

Screenshot 2024-10-02 at 12 54 55 AM

### TABLET

Screenshot 2024-10-02 at 12 55 06 AM

### LAPTOP

Screenshot 2024-10-02 at 12 55 46 AM

### BIG PC

Screenshot 2024-10-02 at 12 56 19 AM

### CODE REVIEW

File: screens/home/youtube-panel/youtube-panel.module.css [hunk 1]
```diff
@@ -1,7 +1,19 @@
 .card {
+  width: 20%;
```
Comment (petrvecera) [inline screens/home/youtube-panel/youtube-panel.module.css:2]: Awesome this looks good however I am not sure if the default size in the % is good. In the desktop view - I would like to have it just 2 lines 5 by 5, right now it moves 2 of them bellow. Maybe we just need to tweek this %? 

I am not sure if we want to 

image

**Review by petrvecera — CHANGES_REQUESTED** [review]

**Comment by petrvecera** [comment]
image
Btw how to to test the big screen even on a small laptop - you can set view to resnponsive, give it big width and set it to 50%.

File: screens/home/youtube-panel/youtube-panel.module.css (hunk 1 above)
Comment (RajVerma97) [reply screens/home/youtube-panel/youtube-panel.module.css]: done

**Review by petrvecera — APPROVED** [review]
nice 👍

**Pull request merged**